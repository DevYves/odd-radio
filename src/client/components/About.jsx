import React, { Component } from 'react';

class About extends Component {
  render() {
    return (
      <div className="container masthead-info">
        <div className="row center">
          <h5>About oddrad.io</h5>
        </div>
        <div className="row">
          <p>oddrad.io is a curated collection of some of the best independent radio stations that Canada has to offer. Each station listed here is powered by volunteers from the station's local community. These DJs pick all their own music, be prepared to hear all kinds of weird stuff.</p>
          <p>The purpose of this app is to collect these unique radio station streams in single place where you can easily navigate from one to another and save your favourites to the presets. You won't love what you hear on one station 100% of the time, but there's always another station to check out. Hit that seek button.</p>
          <p>Unlike other internet radio aggregators, the selection on oddrad.io is kept intentionally small. With less than 30 stations to pick from, oddrad.io recreates a true radio dial experience online. Except all the stations have unique, human curated offerings.</p>
        </div>
      </div>
    )
  }
}

export default About;