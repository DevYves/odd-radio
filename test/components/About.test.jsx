import React from 'react';
import renderer from 'react-test-renderer';
import Enzyme, { shallow } from 'enzyme';
import Adapter from 'enzyme-adapter-react-16';

// React 16 Enzyme adapter
Enzyme.configure({ adapter: new Adapter() });

import About from '../../src/client/components/About';

describe('About', () => {

  const subject = <About />;

  it('matches snapshot', () => {
    expect(renderer.create(subject)).toMatchSnapshot();
  });
});