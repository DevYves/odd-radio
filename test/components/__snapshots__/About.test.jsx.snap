// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About matches snapshot 1`] = `
<div
  className="container masthead-info"
>
  <div
    className="row center"
  >
    <h4>
      About oddrad.io
    </h4>
  </div>
  <div
    className="row"
  >
    <p>
      oddrad.io is a curated collection of some of the best independent radio stations that Canada has to offer. Each station listed here is powered by volunteers from the station's local community. These DJs pick all their own music, be prepared to hear all kinds of weird stuff.
    </p>
    <p>
      The purpose of this app is to collect these unique radio station streams in single place where you can easily navigate from one to another and save your favourites to the presets. You won't love what you hear on one station 100% of the time, but there's always another station to check out. Hit that seek button.
    </p>
    <p>
      Unlike other internet radio aggregators, the selection on oddrad.io is kept intentionally small. With less than 30 stations to pick from, oddrad.io recreates a true radio dial experience online. Hand selected music for your eager ears.
    </p>
  </div>
</div>
`;
